ReactQt/runtime/src/qml/ReactSwitch.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import React 0.1 as React
+import QtQuick.Controls.impl 2.2
```

Comment:
pretty cool! Seems we should use similar approach to land others controls with default styling, but it seems I was not able to find any documentation of QtQuick.Controls.impl

Reply:
I haven't seen any documentation also. `QtQuick.Controls.impl` here needed only for default colors constants.
I took that from Switch.qml and SwitchComponent.qml original implementation.

Reply:
Maybe Slider control customization can use something like this